src/controller/org.controller/org.controller.js

```diff
@@ -667,8 +668,8 @@ async function resetSecret (req, res, next) {
       return res.status(404).json(error.userDne(username))
     }
 
-    if ((orgShortName !== requesterShortName || username !== requesterUsername) && !isSecretariat) {
-      logger.info({ uuid: req.ctx.uuid, message: 'The api secret can only be reset by the Secretariat or if the requester is the user.' })
+    if ((orgShortName !== requesterShortName || username !== requesterUsername) && !isSecretariat && !isOrgAdmin) {
+      logger.info({ uuid: req.ctx.uuid, message: 'The api secret can only be reset by the Secretariat or if the requester is the user or Org Admin.' })
       return res.status(403).json(error.notSameUserOrSecretariat())
```

Comment:
Since we're updating logging, should we also update `notSameUserOrSecretariat()` to include the "Org admin" detail?